entity-types/media_streaming-video/definition.yml

```diff
@@ -0,0 +1,28 @@
+domain: MEDIA_STREAMING
+type: VIDEO
+synthesis:
+  rules:
+    # telemetry with Video* eventType
+    - identifier: appName
+      name: appName
+      encodeIdentifierInGUID: false
+      conditions:
+        - attribute: eventType
+          prefix: Video
+      tags:
+        playerName:
+        playerVersion:
+        deviceType:
+        deviceGroup:
+        deviceManufacturer:
+        deviceModel:
+goldenTags:
+  - playerName
+  - playerVersion
+  - deviceType
+  - deviceGroup
+  - deviceManufacturer
+  - deviceModel
+configuration:
+  alertable: true
+  entityExpirationTime: QUARTERLY
```

Comment:
Please let's not default to a months old expiration. Entities should be refreshed or keep alive periodically instead. 

Could we please move this one to a lower expiration?

Reply:
Thank you, I have changed to `EIGHT_DAYS` which is default.